cosmos/dbt/selector.py

```diff
@@ -175,6 +174,8 @@ def filter_nodes(self, nodes: dict[str, DbtNode]) -> set[str]:
 
         elif TAG_SELECTOR in self.node_name:
             tag_selection = self.node_name[len(TAG_SELECTOR) :]
+            # The following normally wuld return 20
+            # now it is returning 210
```

Comment:
Do we need this comment?

Reply:
Thanks for mentioning, @pankajkoti , we certainly don't need this - removed in 59dad89c130dd5b0f6b43e7c82709e9b4428a1bf